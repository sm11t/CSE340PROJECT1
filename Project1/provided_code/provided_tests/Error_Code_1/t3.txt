TASKS
    1 2 3
POLY
    A = x + 2;
    B = x - 1;
    C(Y,Z) = 3;
    A
    
     = x + 2;
    B = x - 1;
    C(Y,Z) = 3;
    D 
    = x + 2;
    A = x + 
    2;
    B 
    
    = x - 1;
    C
    (Y,Z) = 3;
EXECUTE
    INPUT V;
    w = B(V);
    OUTPUT w;
INPUTS
    3 4 5