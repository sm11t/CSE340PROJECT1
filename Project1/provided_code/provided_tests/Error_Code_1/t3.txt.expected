Semantic Error Code 1: 7 10 11 14 16 19
